<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Microscopic image stack fusion system</string>
  </property>
  <widget class="QLabel" name="video_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>35</y>
     <width>781</width>
     <height>541</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="exp_control">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>233</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="btn_Auto_Exposure">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>203</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Exposure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>263</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure Time:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="btn_Auto_Gain">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>326</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Gain</string>
   </property>
  </widget>
  <widget class="QSlider" name="gain_control">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>356</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>386</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain:</string>
   </property>
  </widget>
  <widget class="QLabel" name="gain_show">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>388</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>183</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Exposure</string>
   </property>
   <widget class="QLabel" name="exp_show">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>308</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Gain</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera</string>
   </property>
   <widget class="QPushButton" name="btn_begin_caputre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start caputre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop_caputre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>57</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop caputre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Video information:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Video captured:</string>
    </property>
   </widget>
   <widget class="QLabel" name="video_inf">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>107</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="video_num">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>128</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>431</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Save Picture</string>
   </property>
   <widget class="QPushButton" name="btn_begin_save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>67</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop_save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>98</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>34</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Interval:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="interval_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_save_path_state">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Save path is not set</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>11</y>
     <width>801</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string>Screen</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>14</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Lens Control</string>
   </property>
   <widget class="QLineEdit" name="com_input_edit">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>24</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>26</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Iuput COM:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>81</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_show_com">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>81</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Not connected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_begin_connect_len">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>108</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Firmware Version:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_firmware_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>147</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_temp_show">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>146</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>147</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+ Diopter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>193</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>- Diopter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>214</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lens SN:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_posi_dip_show">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_neg_dip_show">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_lens_SN_show">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>268</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Diopter</string>
   </property>
   <widget class="QLineEdit" name="diopter_input_edit">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>24</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>27</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Diopter input:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_diopter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start adjusting</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Diopter value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_diop_show">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>78</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Reset_diopter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset diopter</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>423</y>
     <width>151</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Zoom</string>
   </property>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max diopter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>62</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min diopter:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Step:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>121</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>One trip time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_diopter_input">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>28</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_diopter_input">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="step_diopter_input">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>90</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cycle_diopter_input">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>119</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cycle_diopter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>224</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start zoom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Diopter value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_diop_show_2">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>148</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="btn_save_pic_zoom">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>180</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Save pictures</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="btn_save_pic_zoom_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>201</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Back and Forth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>254</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_show_zoom">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>254</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Not in zoom</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_9</zorder>
  <zorder>groupBox_6</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>video_label</zorder>
  <zorder>exp_control</zorder>
  <zorder>btn_Auto_Exposure</zorder>
  <zorder>label</zorder>
  <zorder>btn_Auto_Gain</zorder>
  <zorder>gain_control</zorder>
  <zorder>label_4</zorder>
  <zorder>gain_show</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_7</zorder>
  <zorder>groupBox_8</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
